koa-helmet.d.ts

```diff
@@ -28,10 +28,17 @@ declare namespace koaHelmet {
         mediaSrc?: KoaHelmetCspDirectiveValue[];
         objectSrc?: KoaHelmetCspDirectiveValue[];
         pluginTypes?: KoaHelmetCspDirectiveValue[];
+        prefetchSrc?: KoaHelmetCspDirectiveValue[];
+        reportTo?: string;
         reportUri?: string;
         sandbox?: KoaHelmetCspDirectiveValue[];
         scriptSrc?: KoaHelmetCspDirectiveValue[];
+        scriptSrcAttr?: KoahelmetCspDirectiveValue[];
```

Comment:
nit: This should be `KoaHelmetCspDirectiveValue` with a capital letter for `H` of `Helmet`.